doc/BGP/BGP-supress-fib-pending.md

```diff
@@ -111,7 +111,7 @@ High level requirements:
 Restrictions/Limitations:
 
 - MPLS, VNET routes are out of scope of this document
-- Directly connected and static routes are announced by BGP regardless of their offload status
+- Directly connected are announced by BGP regardless of their offload status
```

Comment:
Directed connected routes?

Reply:
@venkatmahalingam 
```
C>* 100.1.0.0/24 is directly connected, Ethernet160, 00:33:18
```

Reply:
Directly connected **routes** are announced by BGP regardless of their offload status

Reply:
Oh, thanks for catching this!